content/applications/general/users.rst

```diff
@@ -54,37 +51,31 @@ login.
 User type
 ---------
 
-:guilabel:`User Type` can be selected from the :guilabel:`Access Rights` tab of the user form,
-accessible via :menuselection:`Settings app --> Users section --> Manage Users`.
+:guilabel:`User Type` can be chosen on the :guilabel:`Manage Users` page by clicking on the
+:guilabel:`search` bar, and then setting a filter for either :guilabel:`Internal User` or
+:guilabel:`Portal User`. Once that filter is set, click :guilabel:`New` in the upper-left corner to
+create a user account with that setting.
```

Comment:
No need to guilabel "search" bar here, and we can add a reference to the using predefined filters documentation.

Also, I did not notice any change in the configuration when creating a user when a filter is set on this view, I don't think it is correct to state that the user account will be created with the settings of the filter.
```suggestion
:guilabel:`User Type` can be chosen on the :guilabel:`Manage Users` page by clicking on the search
bar, and then :ref:`setting a filter <search/preconfigured-filters>` for either :guilabel:`Internal
User` or :guilabel:`Portal User`.
```